tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/UiValidationSteps.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Utils;
+using OpenQA.Selenium;
+using TechTalk.SpecFlow;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
+{
+    [Binding]
+    internal sealed class UiValidationSteps
+    {
+        private readonly SeleniumContext _seleniumContext;
+
+        public UiValidationSteps(SeleniumContext seleniumContext)
+        {
+            _seleniumContext = seleniumContext;
+        }
+
+        [Then(@"the page contains element with Data ID ([^\s]+)")]
+        public void ThenThePageContainsElementWithDataId(string id)
+        {
+            _seleniumContext.WebDriver.FindElement(By.CssSelector($"[data-test-id={id}]"));
+        }
+
+        [Then(@"element with Data ID ([^\s]+) contains element with Data ID ([^\s]+)")]
```

Comment:
Can you review this method please? The element variable looks redundant.

Reply:
method is updated now to use the element variable